content/_data/changelogs/weekly.yml

```diff
@@ -15605,6 +15605,110 @@
   # pull: 6545 (PR title: Deprecate `Functions#getServerName`)
   # pull: 6548 (PR title: Bump executable-war from 2.4 to 2.5)
 
+  - version: '2.348'
+    date: 2022-05-15
+    changes:
+      - type: rfe
+        category: developer
+        pull: 6538
+        authors:
+          - janfaracik
+        pr_title: Add color override classes
```

Comment:
Developer items need to move to the bottom of the list after the release is complete.